src/lti/tool_config.py

```diff
@@ -25,9 +25,9 @@
 
 NSMAP = {
     'blti': 'http://www.imsglobal.org/xsd/imsbasiclti_v1p0',
-    'xsi': "http://www.w3.org/2001/XMLSchema-instance",
-    'lticp': 'http://www.imsglobal.org/xsd/imslticp_v1p0',
     'lticm': 'http://www.imsglobal.org/xsd/imslticm_v1p0',
+    'lticp': 'http://www.imsglobal.org/xsd/imslticp_v1p0',
+    'xsi': "http://www.w3.org/2001/XMLSchema-instance",
```

Comment:
Not sure this needs to be reordered.

Reply:
Either this needs to be reordered or the test namespaces in the xml in the tests file. That's why #71 is failing. I opted for this because it was easier to see. I can make the changes in the expected xml as well.

Reply:
OK, that works for me!